### DISCUSSION TITLE
Can i Remove default tab in Cstore?

### DISCUSSION THREAD

**Discussion by abdelrhmanyosry** [pimcore/pimcore#17765] [Q&A / Support]
Client don't need the default tab and he asked me if we can disable it or at least remove from the  UI, is it possible?

**Comment by AlephLau** [comment]
Yes, check this out
https://pimcore.com/docs/platform/Pimcore/Extending_Pimcore/Bundle_Developers_Guide/#extending-the-admin-ui
